src/store/hash.js

```diff
@@ -0,0 +1,10 @@
+let isHash = !!location.hash;
+
+const hashState = {
+  setHashState: (state) =>
+    (isHash = typeof state === "boolean" ? state : isHash),
+
+  getHashState: () => isHash,
+};
```

Comment:
개인적으로 아래처럼 만드는게 사용할때 더 깔끔해서 좋다고 느껴져서 남겨봐요 !

```suggestion
const hash = {
  _isHash: !!location.hash,

  set state(state) {
    this._isHash = typeof state === "boolean" ? state : this._isHash;
  },

  get state() {
    return this._isHash;
  },
};

// set state() 실행
// hash.state = true;
```